### PR TITLE
Feature branch: Clear function

### PR DESCRIPTION
**Pull request by orozCoding**
In this branch I created the completed task function and the Clear All Completed functions. Mayor updates are:

- The user can click the checkbox and the task will be marked as completed and the localStorage is updated ✔️ 
- The user is not able to edit the task if it is set as completed ✔️ 
- The user is able to un-check the task and edit it as usual ✔️ 
- The user can click on Clear All Completed and the completed task will be removed and properly updated in localStorage ✔️ 

New functions were created in a new file as requested.

### CODE REVIEW

File: src/completed.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      if (tasks[i].index === index) {
+        tasks[i].completed = true;
+        localStorage.setItem('taskArr', JSON.stringify(tasks));
+      }
+    }
+  } else {
+    for (let i = 0; i < tasks.length; i += 1) {
+      if (tasks[i].index === index) {
+        tasks[i].completed = false;
+        localStorage.setItem('taskArr', JSON.stringify(tasks));
```
Comment (brytebee) [inline src/completed.js:13]: - [ ] **Required:** The data you store to local storage do not persist at the moment, on reload, the data is lost. Let's ensure the data persists. Do consider this [resource for more info](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @orozCoding 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- impressive UI design✔️
- proper use of GitHub flow✔️
- clean code✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **Optional:** Consider adding a live demo link to your Readme.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/completed.js (hunk 1 above)
Comment (orozCoding) [reply src/completed.js:13]: @brytebee Thank you!! Doing it ASAP ✔️

**Review by leolpaz — APPROVED** [review]
Hi @orozCoding , 

Your project is complete!
Congratulations! 🎉🎉 

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by orozCoding** [comment]
@leolpaz Thank you!! ✔️

**Pull request merged**